ext/nnstreamer/extra/nnstreamer_python3_helper.cc

```diff
@@ -480,6 +484,12 @@ parseTensorsInfo (PyObject *result, GstTensorsInfo *info)
     Py_SAFEDECREF (shape_dims);
   }
 
+  /* Validate output tensors info after parsing python object. */
+  if (!gst_tensors_info_validate (info)) {
+    Py_ERRMSG ("Failed to parse the tensors information from python script, it may include invalid tensor type or dimension value.");
```

Comment:
Please print out gst_tensors_info_to_string(info) in the error message. If there is an error in info, it will explain why it is invalid in detail.

Reply:
updated